CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 # Changelog
 
 ## [Unreleased]
+* (x/distriubtion) [#16218](https://github.com/cosmos/cosmos-sdk/pull/16218) Add Autocli config to distribution module.
```

Comment:
nit, can it be under the improvement category?

Reply:
ok ok!!